### PR TITLE
Lesson04 - Assignment

### PR DESCRIPTION
**Pull request by jgalber**
Homepage layout.

### CODE REVIEW

File: lesson04/assignment/css/_layouts.scss [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+@mixin navbar-item {
+    text-decoration: none;
+    color: $white;
+    font-size: $xs-font;
+    transition: all .5s ease-in-out;
+    cursor: pointer;
+}
+
+// Paralax common styling
+@mixin parallax {
```
Comment (tylervanbrocklin) [inline lesson04/assignment/css/_layouts.scss:146]: Great work with the scrolling, looks awesome

File: lesson04/assignment/css/_main.content.scss [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+}
+
+body {
+    @include full-x-y;
+    color: $white;
+}
+
+.Main {
+    // header layout
+    .header {
```
Comment (tylervanbrocklin) [inline lesson04/assignment/css/_main.content.scss:14]: Your SCSS is very clean, good use of nesting

File: lesson04/assignment/css/_layouts.scss (hunk 1 above)
Comment (jgalber) [reply lesson04/assignment/css/_layouts.scss:146]: Thanks!!!

File: lesson04/assignment/css/_main.content.scss (hunk 2 above)
Comment (jgalber) [reply lesson04/assignment/css/_main.content.scss:14]: Thanks Tyler. This time, I tried to polish the code before pushing into the lesson's branch.

**Pull request merged**